src/sas/qtgui/Perspectives/Fitting/media/optimizer.rst

```diff
@@ -121,15 +121,21 @@ otherwise it will restart the fit from a random point in the parameter space.
 
 Use ``--fit=lm`` to select the Levenberg-Marquardt fitter from the command line.
 
+Notes
+-----
+
+v0.8.2 Changed from scipy.leastsq to mpfit for better bounds handling. Use
+``--fit=scipy.leastsq`` to restore the previous behaviour.
+
 References
 ----------
 
-.. [Levenberg1944]
+.. [#Levenberg1944]
     Levenberg, K.
-    *Quarterly Journal of Applied Mathematics*
+    *Quarterly Journal of Applied Mathmatics*
```

Comment:
Nope. Also, the Journal is spelled with a lower case L.

Reply:
Another good catch! Also sorted.
(No L to correct)